optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -695,29 +689,6 @@ ui:
   # The URL for Optimize to use to navigate to modeler in the sidebar
   modelerUrl: ${CAMUNDA_OPTIMIZE_CLOUD_MODELER_URL:}
 
-eventBasedProcess:
```

Comment:
General question, do any of these need documenting in the upgrade guide from 8.5 -> 8.6? Or is the expectation that none of them would have been used anyway?

Or should we consider a generic warning about config so that customers are at least prepared if they apply the upgrade?

Reply:
Yes its one of the AC of the [config removal ticket](https://github.com/camunda/camunda-optimize/issues/13375), I think it would be nice to wait until the end and have one PR to add all config related info to the docs/notices